corehq/apps/users/tasks.py

```diff
@@ -290,3 +290,11 @@ def reset_demo_user_restore_task(commcare_user_id, domain):
 def remove_unused_custom_fields_from_users_task(domain):
     from corehq.apps.users.custom_data import remove_unused_custom_fields_from_users
     remove_unused_custom_fields_from_users(domain)
+
+
+@task()
+def update_domain_date(user_id, domain):
+    from corehq.apps.users.models import WebUser
+    user = WebUser.get_by_user_id(user_id, domain)
+    domain_membership = user.get_domain_membership(domain)
+    domain_membership.last_accessed = datetime.today().date()
```

Comment:
I don't think I need to call `domain_membership.save()`. I don't see it being called anywhere else.

And good point about `user.get_domain_membership(domain)`. I can probably handle that by returning instead of letting it fail.

Reply:
Oh right, `DomainMembership` is a nested schema element in the user, so it cannot be saved directly. However, the user will need to be saved to persist this change. I think this task should call `user.save()`—does that seem right to you?

Reply:
Yes, you're absolutely right! Updated.